### PR TITLE
Tictactoe

### PR DESCRIPTION
**Pull request by lxmarinkovic**

### CODE REVIEW

**Comment by lxmarinkovic** [comment]
#70

File: src/content/learn/tutorial-tic-tac-toe.md [hunk 1]
```diff
@@ -2725,20 +2735,20 @@ export default function Game() {
...
+### Završetak {/*wrapping-up*/}
 
-Congratulations! You've created a tic-tac-toe game that:
+Čestitamo! Napravili ste igru iks-oks koja:
 
-- Lets you play tic-tac-toe,
-- Indicates when a player has won the game,
-- Stores a game's history as a game progresses,
-- Allows players to review a game's history and see previous versions of a game's board.
+- Omogućava igranje iks-oks,
```
Comment (rammba) [inline src/content/learn/tutorial-tic-tac-toe.md:2744]: Omogućava igranje iks-oks =>  Omogućava igranje iks-oks `igre`

File: src/content/learn/tutorial-tic-tac-toe.md [hunk 2]
```diff
@@ -2725,20 +2735,20 @@ export default function Game() {
...
+- Indikuje kada je igrač pobedio u igri,
```
Comment (rammba) [inline src/content/learn/tutorial-tic-tac-toe.md:2745]: Indikuje => Označava

File: src/content/learn/tutorial-tic-tac-toe.md [hunk 3]
```diff
@@ -2907,12 +2917,12 @@ body {
...
-If you have extra time or want to practice your new React skills, here are some ideas for improvements that you could make to the tic-tac-toe game, listed in order of increasing difficulty:
+Ako imate dodatno vreme ili želite da vežbate svoje nove React veštine, evo nekoliko ideja za unapređenja igre iks-oks, poređanih po rastućem nivou težine:
 
-1. For the current move only, show "You are at move #..." instead of a button.
-1. Rewrite `Board` to use two loops to make the squares instead of hardcoding them.
-1. Add a toggle button that lets you sort the moves in either ascending or descending order.
-1. When someone wins, highlight the three squares that caused the win (and when no one wins, display a message about the result being a draw).
-1. Display the location for each move in the format (row, col) in the move history list.
+1. Samo za trenutni potez prikažite poruku "Nalazite se na potezu #..." umesto dugmeta.
+2. Prepišite component-u `Board` da koristi dve petlje za kreiranje kvadrata umesto da ih hardkodirate.
```
Comment (rammba) [inline src/content/learn/tutorial-tic-tac-toe.md:2923]: hardkodirate => hardcode-ujete

**Review by rammba — COMMENTED** [review]
Hey @lxmarinkovic, first of all thanks for this awesome contribution. 🚀  You've made a great job translating this oversized file, probably the biggest in the whole documentation. I can just imagine how much time it would take. ⌚ 

I've added bunch of comments, but most of them are some small typos or better terms than those you used. You really did an awesome thing, regardless of that.

I just want to ask you not to merge future PRs without someone's approval, not because your translation doesn't fit, but because two pair of eyes are always better than one.

Thanks for this once more, you can address comments in another PR or we can go through them together if you think something is off. I also spent quite some time on reviewing, so maybe I missed something as well. 😄

File: src/content/learn/tutorial-tic-tac-toe.md [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
 
-You will build a small tic-tac-toe game during this tutorial. This tutorial does not assume any existing React knowledge. The techniques you'll learn in the tutorial are fundamental to building any React app, and fully understanding it will give you a deep understanding of React.
+Tokom ovog tutorijala napravićete jednostavnu igru iks-oks. Ovaj tutorijal ne zahteva prethodno znanje o React-u. Tehnike koje ćete naučiti su fundamentalne za pravljenje bilo koje React aplikacije, a potpuno razumevanje ovog tutorijala će vam pružiti duboko razumevanje React-a.
 
 </Intro>
 
 <Note>
 
-This tutorial is designed for people who prefer to **learn by doing** and want to quickly try making something tangible. If you prefer learning each concept step by step, start with [Describing the UI.](/learn/describing-the-ui)
+Ovaj tutorijal je dizajniran za one koji preferiraju da ***uče kroz praksu*** i žele brzo da probaju napraviti nešto konkretno. Ukoliko više volite da učite svaki koncept korak po korak, za početak uzmite [Opisivanje UI-a](/learn/describing-the-ui)
```
[the remark this answers is not in the document]
Comment (lxmarinkovic) [reply src/content/learn/tutorial-tic-tac-toe.md:13]: Thanks for taking a look at it! 😀 Yes, I know it needs a pair of fresh eyes, but I think we should focus on the big bulk in front of us until we reserve the domain, and then we can do all necessary fixes, e.g., typos, etc. What do you think about putting a strategy like that 🤔

[the remark this answers is not in the document]
Comment (rammba) [reply src/content/learn/tutorial-tic-tac-toe.md:13]: Yeah, I think it's a good strategy for now. At least, we have these comment here, and any others on future PRs, so we can focus on them after securing domain. I think we have a same goal now 😉

**Pull request merged**